### PR TITLE
Document for user guide

### PR DESCRIPTION
**Pull request by keshavsoni251199**
This pull request contain files and figures for user guide.

### CODE REVIEW

File: docs/source/Userguide.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+---------------------------------------------
+**Login Page:** User have to login with their credentials to access FIWARE Lab dashboard.
+
+.. figure:: figures/login-page.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**API Access Page:** User can find Project ID after clicking view credentials in API Access section of FIWARE Lab dashboard.
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: This information is not required by the user, so we can remove it

File: docs/source/Userguide.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+.. figure:: figures/view-credentials.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Instances:** User have to create instances in instances tab of FIWARE Lab dashboard.
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: Mention the number of instances and its configuration for K8s setup

File: docs/source/Userguide.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+**Overview Page:** User have to click on kubernetes button to redirect to kubernetes GUI for cluser deployment.
+
+.. figure:: figures/overview-page.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+Pages in the kubernetes GUI:
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: Change heading to Kubernetes support UI

File: docs/source/Userguide.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+**Components details:** User will get option to View and edit components. For this first user needs to select the Cluster name from the Cluster List and after that user can view/edit its components.
+
+.. figure:: figures/component.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Add Node:** User can add node details in "Add Node" option after clicking on the ID shown in the Dashboard. User need to provide details such as VM username (centos/ubuntu), VM IP (internal IP), pem file to access the VM and after submitting the information, VM details will be added to the cluster.
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: Add screenshot with the Add node option.

File: docs/source/Userguide.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+.. figure:: figures/add-node.png
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: Add some IP address in the screenshot and remove 111

File: docs/source/Userguide.rst [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Add Components:** User can add Fiware GEs to the cluster by using "Add Components" option after clicking on the ID  shown in the Dashboard. User can select the components which he needs in the cluster by selecting the Enable option. User can also select the version of the Fiware GEs and its database version from the dropdown. After submitting the details, Fiware GEs will be added to the cluster.
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: Add screenshot with Add Components option

File: docs/source/Userguide.rst [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+**Delete Cluster:** User will get the option to delete the cluster in GUI. After clicking on icon it will ask to confirm if you want to delete the cluster.
+
+.. figure:: figures/delete-cluster.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Note:** UI may change to match user requirement.
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: Remove this statement

File: docs/source/Userguide.rst [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+**Advantages:**
+
+1. Easier to create and manage kubernetes cluster through GUI.
+
+2. User will be provided with option to deploy Fiware GEs through GUI.
+
+3. User can select Fiware GEs from the catalogue and those GEs will be deployed.
+
+**Limitations:**
```
Comment (pooja1pathak) [inline docs/source/Userguide.rst]: Remove Limitation, as they no longer exist for user

**Comment by pooja1pathak** [comment]
Add Userguide.rst to docs folder and also create one step by step document for creating a cluster with screenshots mentioning step1, step2 , and so on.

File: docs/source/Userguide.rst (hunk 2 above)
Comment (keshavsoni251199) [reply docs/source/Userguide.rst]: Mentioned the required configuration for setup.

File: docs/source/Userguide.rst (hunk 4 above)
Comment (keshavsoni251199) [reply docs/source/Userguide.rst]: Screenshot was added to guide user to add node.

File: docs/source/Userguide.rst (hunk 5 above)
Comment (keshavsoni251199) [reply docs/source/Userguide.rst]: Screenshot with Ip address was updated in file.

File: docs/source/Userguide.rst (hunk 6 above)
Comment (keshavsoni251199) [reply docs/source/Userguide.rst]: Screenshot was added to guide user to add component.

File: docs/source/Userguide.rst (hunk 7 above)
Comment (keshavsoni251199) [reply docs/source/Userguide.rst]: The changes were done.

File: docs/source/Userguide.rst (hunk 3 above)
Comment (keshavsoni251199) [reply docs/source/Userguide.rst]: Changes were done.

File: docs/source/Userguide.rst (hunk 8 above)
Comment (keshavsoni251199) [reply docs/source/Userguide.rst]: Changes were done in the document.

File: docs/Userguide.rst [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+---------------------------------------------
+**Login Page:** User have to login with their credentials to access FIWARE Lab dashboard.
+
+.. figure:: source/figures/login-page.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Instances:** User have to create two instances in instances tab of FIWARE Lab dashboard with minimum configuration flavor medium(2VCPUS).
```
Comment (pooja1pathak) [inline docs/Userguide.rst]: change statement as:
User have to create two instances in instances tab of FIWARE Lab dashboard with configuration as follows:
Flavor: medium or large
OS: Centos

**Review by pooja1pathak — APPROVED** [review]

File: docs/Userguide.rst (hunk 9 above)
Comment (keshavsoni251199) [reply docs/Userguide.rst]: Changes were done

**Pull request merged**